s6/root/etc/logrotate.d/nginx

```diff
@@ -0,0 +1,14 @@
+/config/log/nginx/*.log {
+        weekly
+        rotate 14
+        compress
+        delaycompress
+        nodateext
+        notifempty
+        missingok
+        sharedscripts
+        postrotate
+                s6-svc -h /var/run/s6/services/nginx
```

Comment:
maybe it should be aligned with other lines

Reply:
nope, it's a command to run after log rotate. I'll remove nginx for now